codecov.yaml

```diff
@@ -2,15 +2,3 @@
 codecov:
   ci:
     - source.cloud.google.com
-
-# exclude all autogenerated code
-ignore:
- - google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/*/v2/BaseBigtable*.java
- - google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/*/v2/stub/GrpcBigtable*Stub.java
- - google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/*/v2/stub/GrpcBigtable*CallableFactory.java
- - grpc-google-cloud-bigtable-admin-v2
- - proto-google-cloud-bigtable-admin-v2
- - grpc-google-cloud-bigtable-v2
- - proto-google-cloud-bigtable-v2
- - .kokoro
- - .github
```

Comment:
Why isnt this respecting the exclusion?
https://github.com/googleapis/java-bigtable/blob/master/synth.py#L45

Reply:
I'm going to try closing and allowing synth to regen; this may have been done just before you merged in that exclusion